gulpfile.js

```diff
@@ -0,0 +1,103 @@
+var gulp = require('gulp');
+var path = require('path');
+var concat = require('gulp-concat');
+var minifyCss = require('gulp-minify-css');
+var gutil = require('gulp-util');
+var argv = require('yargs')
+    .usage(
+      'Usage: $0 --appengine [Path to app engine]' +
+      '--clear_datastore [clear all data storage]' +
+      '--enable_sendmail [send emails]' +
+      '--minify [minify only in production]').argv;
+var gulpStartGae = require('./scripts/gulp-start-gae-devserver');
+var manifest = require('./manifest.json');
+
+var serveCmd = argv.appengine;
```

Comment:
Try to use the same name throughout for the same concept. In this case it should be something like "var x = argv.x" (I mean, not literally 'x', but the x's should preferably be the same).